src/Polly.Core/Builder/ResilienceStrategyOptions.cs

```diff
@@ -0,0 +1,21 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace Polly.Builder;
+
+/// <summary>
+/// The options associated with the <see cref="IResilienceStrategy"/>.
+/// </summary>
+public class ResilienceStrategyOptions
+{
+    /// <summary>
+    /// Gets or sets the name of the strategy.
```

Comment:
Let's also add the intended usage of these two properties (metering and logging).